.travis.yml

```diff
@@ -1,10 +1,13 @@
 sudo: required
 services: docker
-script: docker build . -t cs50/cli
+script: docker build . -t cs50/cli:${TRAVIS_BRANCH/master/latest}
```

Comment:
What's this syntax do?

Reply:
If branch name is `master`, use `latest` as tag name. Otherwise, use name of the branch. Branch name has to be specified in the condition in the line right below that one (i.e., not all branches are pushed).

Reply:
fancy